dotnet/Dockerfile.tpl

```diff
@@ -0,0 +1,27 @@
+# AUTOGENERATED FILE
+FROM #{FROM}
+
+RUN [ "cross-build-start" ]
+
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 gettext icu-devtools
```

Comment:
can you pass extra option `--no-install-recommends` to the `apt-get install` command?

Reply:
and `rm -rf /var/lib/apt/lists/*` as well

Reply:
the `rm -rf` was in the next run command, so I merged them up